File: SCALA/fizzbuzzscala/src/main/scala/FizzBuzz.scala
object FizzBuzz extends App {
  def FizzBuzzConverter(x: Int) = {
    if(x % 15 == 0)
      "FizzBuzz"
    else if(x % 5 == 0)
      "Buzz"
    else if(x % 3 == 0)
      "Fizz"    
  }
}

File: ELIXIR/fizzbuzz/lib/fizzbuzz.ex
defmodule Fizzbuzz do
  def fizzbuzzer(user_input) do
    cond do
      rem(user_input,3) == 0 && rem(user_input,5) == 0 ->
        "FizzBuzz"
      rem(user_input,3) ->
        "Fizz"
    end
  end
end
